src/viking.js

```diff
@@ -1,11 +1,84 @@
 // Soldier
-class Soldier {}
+class Soldier {
+    constructor(health, strength) {
+        this.health = health;
+        this.strength = strength
+    }
+    attack() {
+        return this.strength
+    }
+    receiveDamage(damage) {
+        const lifePoints = this.health -= damage;
+    }
+}
 
 // Viking
-class Viking {}
+class Viking extends Soldier {
+    constructor(name, health, strength) {
+        super(health, strength)
+        this.name = name;
+    }
+    receiveDamage(damage) {
+        const lifePoints = this.health -= damage;
```

Comment:
hmmm esa expresion no es del todo correcta, simplemente tienes que hacer un` this.health -= damage`

Reply:
te hecho no vuelvas a usar lifePoints nunca mas, asi que solo preocupate de restarle a this.health el damage que sea